Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -25,84 +25,84 @@
 bool OpenALStream::Start()
 {
   m_run_thread.Set();
-  bool bReturn = false;
+  bool b_return = false;
```

Comment:
`b` prefix was used here as hungarian notation. IMO this should be renamd to something like `success` or w/e. Sorry for nitpick :(